src/components/Table/Table.scss

```diff
@@ -99,15 +82,14 @@
   padding: 0;
 
   // Empty checkbox.
-  &::before {
-    border: 1px solid $ms-color-neutralTertiary;
-    content: '';
-    display: block;
-    height: 14px;
-    left: 2px;
+  &::after {
+    @include ms-Icon;
+    content: '\e739';
```

Comment:
Same here, try using `@include ms-Icon--Checkbox` to bring in the content value.